File: Service_eHospital_DongNai_A/Implement/zzz_NoiTru_XuatLaiXMLService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class zzz_NoiTru_XuatLaiXMLService : BaseService<zzz_NoiTru_XuatLaiXML, Izzz_NoiTru_XuatLaiXMLRepository>
    , Izzz_NoiTru_XuatLaiXMLService
    {
    private readonly Izzz_NoiTru_XuatLaiXMLRepository _zzz_NoiTru_XuatLaiXMLRepository;
    public zzz_NoiTru_XuatLaiXMLService(Izzz_NoiTru_XuatLaiXMLRepository zzz_NoiTru_XuatLaiXMLRepository) : base(zzz_NoiTru_XuatLaiXMLRepository)
    {
    _zzz_NoiTru_XuatLaiXMLRepository = zzz_NoiTru_XuatLaiXMLRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/zzz_NoiTru_XuatLaiXML.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { zzz_NoiTru_XuatLaiXML } from './zzz_NoiTru_XuatLaiXML.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class zzz_NoiTru_XuatLaiXMLService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'STT', 'SoBenhAn'];

    List: zzz_NoiTru_XuatLaiXML[] | undefined;
    ListFilter: zzz_NoiTru_XuatLaiXML[] | undefined;
    FormData!: zzz_NoiTru_XuatLaiXML;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "zzz_NoiTru_XuatLaiXML";
    }
}

